src/renderer/qssWidget/js/qssVolumeWidget.js

```diff
@@ -444,19 +446,24 @@
      * subtracted from or added to the setting's value.
      * @return {Boolean} Whether there was a change in the setting's value.
      */
-    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier) {
-        var step = schema.divisibleBy * stepMultiplier;
+    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier, previousValue) {
+        var restrictedValue;
+        if (value === 0 && previousValue) {
```

Comment:
Be clearer here about whether the expectation is that previousValue is being checked for being !== undefined or 0 (falsy), or just !== undefined